src/sdk/pynni/nni/counter.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
better to put this file under https://github.com/microsoft/nni/tree/master/src/sdk/pynni/nni/compression/torch/utils